source/NPC.cpp

```diff
@@ -229,9 +234,10 @@ void NPC::Do(const ShipEvent &event, PlayerInfo &player, UI *ui, bool isVisible)
 				if(bay.ship)
 					actions[bay.ship.get()] |= event.Type();
 			
-			// If a mission ship is captured, let it live on under its new
-			// ownership but mark our copy of it as destroyed.
-			if(event.Type() & ShipEvent::CAPTURE)
+			// If a mission ship without the "capture" tag is captured, let it
+			// live on under its new ownership but mark our copy of it as destroyed.
+			auto it = actions.find(ship.get());
+			if(event.Type() & ShipEvent::CAPTURE & (it->second != ShipEvent::CAPTURE))
```

Comment:
The second & would need to be a logical &. Otherwise you'd just doing a bitwise and and counting on ShipEvent::CAPTURE being an odd number.